src/dmd/cppmangle.d

```diff
@@ -584,7 +584,20 @@ private final class CppMangleVisitor : Visitor
              */
             TemplateInstance ti = d.parent.isTemplateInstance();
             assert(ti);
-            source_name(ti);
+            Dsymbol p = ti.toParent();
+            if (p && !p.isModule() && tf.linkage == LINK.cpp)
+            {
+                buf.writeByte('N');
+                CV_qualifiers(d.type);
+                prefix_name(p);
+                if (d.isDtorDeclaration())
+                    buf.writestring("D1");
```

Comment:
I thought we don't merge PRs anymore if the introduce uncovered code (and don't have a good reason to do so)?

Reply:
If it was refactored to be common code, then it would show as being  covered.